package.json

```diff
@@ -25,7 +25,8 @@
     "postinstall": "node scripts/npm/postinstall",
     "release": "./scripts/travis/release.sh",
     "build": "gulp build",
-    "test": "gulp test && node test/accessibility/wcag.js"
+    "test": "gulp test && node test/accessibility/wcag.js",
```

Comment:
We have `gulp test:a11y` in cfgov-refresh.. we should move that here too at some point so everything is under `gulp test`.